src/components/Button.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const widthC = {
+    fifty: '50%',
+    half: '25%',
+  };
+  let widthB;
+  /* eslint-disable */
+  if (props.wide === true) {
+    widthB = widthC.fifty;
+  } else {
+    widthB = widthC.half;
+  }
+  return (
+    <button className="button-button" style={{ backgroundColor: props.color, width: widthB }} type="button">
+      {props.name}
+    </button>
+  );
+};
+/* eslint-enable */
```

Comment:
* [ ] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))